.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,49 @@
+---
+name: 🐛 Bug Report
+about: If something isn't working as expected 🤔
+
+---
+
+<!-- Thanks for helping grpc-swift! Before you submit your issue, please make sure to check the following boxes by putting an x in the [ ] (don't: [x ], [ x], do: [x]) -->
+
+### New Issue Checklist
+
+- [ ] Updated grpc-swift to the latest version
+- [ ] I read the [Contribution Guidelines](https://github.com/grpc/grpc-swift/blob/master/CONTRIBUTING.md)
+- [ ] I searched for [existing GitHub issues](https://github.com/grpc/grpc-swift/issues)
```

Comment:
Idea: specifically mention that several SSL issues have been discussed (and solved) in previous issues?

Reply:
I added a link for resolved SSL issues :)